blueprints/app/files/_js_eslint.config.mjs

```diff
@@ -16,7 +16,7 @@ import globals from 'globals';
 import js from '@eslint/js';
 
 import ember from 'eslint-plugin-ember/recommended';
-import prettier from 'eslint-plugin-prettier/recommended';
+import eslintConfigPrettier from 'eslint-config-prettier';
```

Comment:
@bertdeblock why would we still need this if prettier runs outside of ESLint?

Reply:
ah, right, so it's to disable any stylistic related rules in ESLint to avoid conflicts